{
  "debianbug": 864859,
  "description": "Jython before 2.7.1rc1 allows attackers to execute arbitrary code via a crafted serialized PyFunction object.",
  "releases": {
    "bullseye": {
      "fixed_version": "2.5.3-17",
      "repositories": {
        "bullseye": "2.7.2+repack1-3"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "buster": {
      "fixed_version": "2.5.3-17",
      "repositories": {
        "buster": "2.7.1+repack1-4~deb10u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "sid": {
      "fixed_version": "2.5.3-17",
      "repositories": {
        "sid": "2.7.2+repack1-3"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    },
    "stretch": {
      "fixed_version": "2.5.3-16+deb9u1",
      "repositories": {
        "stretch": "2.5.3-16+deb9u1",
        "stretch-security": "2.5.3-16+deb9u1"
      },
      "status": "resolved",
      "urgency": "not yet assigned"
    }
  },
  "scope": "local"
}